### ISSUE TITLE
Non-Conforming default values with decimal/currency

### ISSUE THREAD

**Issue by CarsonStiles** [RobinHerbots/Inputmask#1274]
I am attempting to format currency input elements that have a non-conforming default value identifying the type of amount to be entered. I am not using labels as the default value of the input is used as a tag in a find/replace script to update a textarea element. What I found is this works for the first element, but all remaining elements do not receive the formatting mask. Once the initial value is removed the formatting works as expected. Are non-conforming default values allowed, and if not, is it possible to ignore the formatting mask on the initialization of the element?

I am using jquery.inputmask bundle version 3.3.1 

$('.amount').inputmask('decimal', {
      radixPoint: '.',
      placeholder: '0',
      groupSeparator: ',',
      digits: 2,
      digitsOptional: !1,
      autoGroup: true,
      prefix: '$',
                clearIncomplete: false
});

<input type="text" class="amount" id="Amount1" value="<<PAY AMT>>"/>
<input type="text" class="amount" id="Amount2" value="<<AMT REC>>"/>

**Comment by RobinHerbots** [comment]
@CarsonStiles , 

Non-conforming default values are allowed, saying the inputmask tries to apply the mask on it.

Can you give some examples of the default values used.

**Comment by CarsonStiles** [comment]
Below are the input statements I am using for testing. I have tried initializing by both the class and the id with the same results. I don't believe the inputmask attempts to apply the mask on the default value, but something is preventing the mask from being applied to the second element with the cause being related to the default value.

`<input type="text" class="amount" id="Amount1" name="AmountTest1" value="<<PAY AMT>>"/>
<input type="text" class="amount" id="Amount2" name="AmountTest2" value="<<AMT REC>>"/>`
